.github/workflows/vercel-deploy.yml

```diff
@@ -21,5 +21,5 @@ jobs:
                   vercel-token: ${{secrets.VERCEL_TOKEN}}
                   vercel-org-id: ${{secrets.VERCEL_ORG_ID}}
                   vercel-project-id: ${{secrets.VERCEL_PROJECT_ID}}
-                  vercel-args: '--prod --build-env NPM_AUTH=${{ secrets.ARTIFACTORY_NPM_AUTH }} --build-env NPM_EMAIL=${{ secrets.ARTIFACTORY_NPM_EMAIL }} --build-env STRAPI_URL=${{ secrets.STRAPI_DEV_URL }}'
+                  vercel-args: '--prod --build-env NPM_AUTH=${{ secrets.ARTIFACTORY_NPM_AUTH }} --build-env NPM_EMAIL=${{ secrets.ARTIFACTORY_NPM_EMAIL }} --build-env STRAPI_URL=${{ secrets.STRAPI_DEV_URL }} --build-env REALM_API_URL=${{ secrets.REALM_API_URL }} --build-env REALM_SEARCH_URL=${{ secrets.REALM_SEARCH_URL }}'
```

Comment:
If you want this to work in Vercel, you might be able to do `--build-env HOST_URL=${{ secrets.VERCEL_URL }}`, to set the HOST_URL. Mentioning since I saw that the preview environment had a 500.

Reply:
ahhh ok that makes sense, I was just testing on Vercel and was confused about this.

Reply:
Ah that's unfortunate. Kind of ugly, but you can do something like https://github.com/mongodb/devcenter/blob/main/next.config.js#L5.

Reply:
Ahh I see, I didn't realize `VERCEL_URL` was already populated by Vercel's deployment, but I think the logic we do in the config should work!